worker/src/dns.rs

```diff
@@ -77,6 +77,7 @@ pub fn build_dns_vc(pk: &JWK, domain: String) -> Result<Credential> {
                           "@id": "https://tzprofiles.com/timestamp",
                           "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
                       },
+                      "dnsServer": "https://tzprofiles.com/dns-server",
```

Comment:
```suggestion
                      "dnsServer": "https://tzprofiles.com/dnsServer",
```
Convention for IRIs seems to be lowerCamelCase for properties and UpperCamelCase for things.